StortSpelProjekt/Game/src/main.cpp

```diff
@@ -22,6 +22,8 @@ Scene* GameScene(SceneManager* sm);
 Scene* ShopScene(SceneManager* sm);
 
 void GameInitScene(Scene* scene);
+void ShopInitScene(Scene* scene);
+void ParticleInit();
```

Comment:
I suppose this function is not meant to be used outside of this class, so private would make more sence.